### ISSUE TITLE
PF background images affecting bundle sizes

### ISSUE THREAD

**Issue by ahileseverquote** [patternfly/patternfly-design#1202]
Hi. I've noticed when building React apps using Patternfly with Webpack that there are warnings related to asset sizes on the PF background images. 

When building bundles, I am seeing the following warnings:

```
WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets:
  images/pfbg_768@2x.jpg (354 KiB)
  images/pfbg_2000.jpg (270 KiB)
  images/pfbg_992@2x.jpg (575 KiB)
```

I'm using the same `url-loader` plugin config in my Webpack configuration as what's used in the [patternfly-react-seed](https://github.com/patternfly/patternfly-react-seed/blob/master/webpack.common.js) project.

```
{
          test: /\.(jpg|jpeg|png|gif)$/i,
          include: [
            path.resolve(__dirname, 'src'),
            path.resolve(__dirname, 'node_modules/patternfly'),
            path.resolve(__dirname, 'node_modules/@patternfly/patternfly/assets/images'),
            path.resolve(__dirname, 'node_modules/@patternfly/react-styles/css/assets/images'),
            path.resolve(__dirname, 'node_modules/@patternfly/react-core/dist/styles/assets/images'),
            path.resolve(__dirname, 'node_modules/@patternfly/react-core/node_modules/@patternfly/react-styles/css/assets/images'),
            path.resolve(__dirname, 'node_modules/@patternfly/react-table/node_modules/@patternfly/react-styles/css/assets/images'),
            path.resolve(__dirname, 'node_modules/@patternfly/react-inline-edit-extension/node_modules/@patternfly/react-styles/css/assets/images')
          ],
          use: [
            {
              loader: 'url-loader',
              options: {
                limit: 5000,
                outputPath: 'images',
                name: '[name].[ext]',
              }
            }
          ]
        }
```

Module versions in my package.json:
- @patternfly/patternfly: ^4.90.5
- @patternfly/react-core: ^4.101.3
- @patternfly/react-icons: ^4.9.9

Are there any further optimisation steps that can be taken for PF images when building with Webpack? I've taken a look through historical issues on this project and cannot find anything. There looks to have been similar issues with the [react-icons lib](https://github.com/patternfly/patternfly-react/issues/3268) but those have been resolved. Any info/guidance would be appreciated as these feel like pretty sizeable chunks to be having in Production builds. Thanks in advance 🙂

**Comment by mceledonia** [comment]
@mcoker What would be the best way to implement this with an SVG? I'm assuming we can export the logo file and have it positioned in the browser, freeing up some flexibility for resizing/responsiveness. Let me know if there's a better way.

**Comment by mcoker** [comment]
@mceledonia 

#### Login component

Right now we use this [background image component](https://www.patternfly.org/v4/components/background-image/html) + the login component for all of the login component examples. In the react component, a user has to specify their background image to use, and we pass their image to the background image component. If they don't specify one, we don't include the background image component. However the core background image component on its own has the image shown in our examples built in to the component. Users can supply their own image if they want, but we provide the default image. The background image component also comes with an SVG filter/overlay (that is overridable by the user) that, as I remember it, kinda darkens the default background image but still lets the light colors pop. You can see that filter here - https://github.com/patternfly/patternfly/blob/416dcb5e12b562292e4eb1538cbb6b99c82c9396/src/patternfly/components/BackgroundImage/background-image.hbs#L6-L20

At a minimum, we could keep all of that as it is and just remove the default image and the filter the background image component ships with, and require the image used to be supplied by the user. In our examples, we could then pass the PF SVG you're showing here. You could provide the SVG however you like and we'll add it in as an asset just for our examples, and won't ship with the core or react packages.

Other questions with this update if we want to take them into consideration:
* Where does the background color come from? Should that be built into the SVG and/or the background image component? We can support both.
  * Also taking into consideration that the background image component can be used anywhere, not just with the login component, although that's the only place we use it within our components currently.
* Would the background image always be affixed/sized to the viewport in a particular way? Currently the background image component:
  * Spans the full height/width of the viewport
  * uses a "fixed" position (as you scroll content on the page, the background doesn't move/scroll with it)
  * the image scales to "[cover](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size)", which will size the shortest side (height or width) of the image down to match the viewport size, then crop off any excess from the larger side, cropping from the top/left corner.

With those, curious if you have any thoughts and if maybe we have other examples of login screens from products to provide insight, or what products might like to ideally see.

#### About modal component

Basically the same as the login page, but we don't use the background image component (the image is part of the about modal component), it comes by default in both core and react, and can be overridden by the user. It uses the same fixed position, scale set to "cover", cropping from the top/left corner. 

I guess it depends on whether we want to continue to ship something generic by default (but use something smaller than we have now, per the original issue), or only use an image specified by the user. If supplied by the user, is it required or optional? If optional, I'm assuming the layout would need to change, too.

**Comment by mceledonia** [comment]
Some updated screens of login + about after discussion with @mcoker @mcarrano 
![image](https://user-images.githubusercontent.com/30935938/207427364-d7983629-b827-402f-a1ff-2dfd00c552ed.png)
![image](https://user-images.githubusercontent.com/30935938/207427405-06d5bdbf-0589-48e9-a132-b17c661afc51.png)

Image should be an SVG positioned to bottom right, if no image default to PF logo.

**Comment by mcarrano** [comment]
Thanks @mceledonia . Have opened https://github.com/patternfly/patternfly/issues/5282 to begin development work and will close this.
